src/shared/components/hmcts-error-summary/hmcts-error-summary.component.html

```diff
@@ -0,0 +1,24 @@
+<div *ngIf="!showWarningMessage" id="errorSummary" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1"
```

Comment:
there is hmcts-error-summary inside the common-lib already, can we start using that?
I have actually remove all the generic components in manage org and move it to common-lib, but the PR in manage org never been merged and now outdated. but may be a good practice to start using the common-lib for now. (and modify the components in common-lib).

Reply:
@elviani I have changed it slightly for handling scenario 429. I will do that later if you alright